samza-core/src/main/scala/org/apache/samza/container/SamzaContainer.scala

```diff
@@ -592,17 +614,25 @@ object SamzaContainer extends Logging {
         new StorageConfig(config).getChangeLogDeleteRetentionsInMs,
         new SystemClock)
 
+      var sideInputStorageManager: SideInputStorageManager = null
```

Comment:
It looks like the pattern for storageManager is to still be non-null even if there are no stores. Should we use a consistent pattern with this?

Reply:
Tracked in SAMZA-1780.